### ISSUE TITLE
Nothing happen when trying to interact with underlaying view controller when LPWebInterstitial banner is presented

### ISSUE THREAD

**Issue by vlad-shostak** [Leanplum/Leanplum-iOS-SDK#523]
## Expected Behavior
Interact with underlaying view controller when LPWebInterstitial banner is presented.

## Actual Behavior

Nothing happen when trying to interact with underlaying view controller when LPWebInterstitial banner is presented. The only way to unfreeze to dismiss banner using close button.

## Steps to Reproduce the Problem

  1. Present LPWebInterstitialViewController not in full screen but as a banner.
  2. Try to interact with underlaying view controller.

## Specifications

  - Version: 4.1.0. This bug appear in version 4.0.0. Previously was ok.
  - Platform: iOS 15.5.

https://user-images.githubusercontent.com/28777985/195603151-0a4cd23c-530d-4f5f-bbb0-eb10e36644cd.mp4

**Comment by nzagorchev** [comment]
Hi @vlad-shostak 

Would it be possible to test using this [branch](https://github.com/Leanplum/Leanplum-iOS-SDK/tree/banner)?

**Comment by vlad-shostak** [comment]
Hi @nzagorchev 

I tried early to change `modalPresentationStyle` but it didn't help. And now I tried again and the bug still reproduce.

**Comment by nzagorchev** [comment]
The changes in the branch mimic the code used before 4.0.0.

Chronologically:
- the `modalPresentationStyle` was changed together with other updates
- I applied this fix #496  - the bug was no longer reproducing on our test applications, however, it seems it solved it only partially
- I was able to reproduce it again yesterday with another sample app, hence the changes in the branch linked above - it `presentOverVisible` and uses `overCurrentContext`.

Using the above branch I no longer reproduce the issue on the sample apps we have.
Let me know if it works with the linked branch, otherwise, we will need a sample application that reproduces the issue.
